roachtest: schemachange/indexrollback/tpcc/w=1000 failed

SHA: https://github.com/cockroachdb/cockroach/commits/04ef15974085e14f758b20c552a84052eac9fa2b

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=schemachange/indexrollback/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1180753&tab=buildLog

```
The test failed on master:
	test.go:1202: test timed out (3h0m0s)
	cluster.go:1267,tpcc.go:142,schemachange.go:375,test.go:1214: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1180753-schemachange-indexrollback-tpcc-w-1000:5 -- ./workload check tpcc --warehouses=1000 {pgurl:1} returned:
		stderr:
		
		stdout:
		I190316 16:58:26.162178 1 workload/tpcc/tpcc.go:292  check 3.3.2.1 took 5.298227003s
		I190316 16:58:45.986489 1 workload/tpcc/tpcc.go:292  check 3.3.2.2 took 19.824234132s
		I190316 16:58:54.614923 1 workload/tpcc/tpcc.go:292  check 3.3.2.3 took 8.62835764s
		I190316 17:03:44.000969 1 workload/tpcc/tpcc.go:292  check 3.3.2.4 took 4m49.385960139s
		I190316 17:07:12.215695 1 workload/tpcc/tpcc.go:292  check 3.3.2.5 took 3m28.214617545s
		: signal: killed

```

SHA: https://github.com/cockroachdb/cockroach/commits/c2939ec9a4f15b7fb8683a5805deeb241953e7aa

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=schemachange/indexrollback/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1182991&tab=buildLog

```
The test failed on master:
	cluster.go:1267,tpcc.go:130,cluster.go:1605,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1182991-schemachange-indexrollback-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=1h30m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		 1208.0   1208.0 orderStatus
		 1h0m36s    20341            0.0           17.2      0.0      0.0      0.0      0.0 payment
		 1h0m36s    20341            0.0            1.8      0.0      0.0      0.0      0.0 stockLevel
		E190318 15:14:38.322748 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.63:26257: connect: connection refused
		_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		 1h0m37s    86809            0.0            1.9      0.0      0.0      0.0      0.0 delivery
		 1h0m37s    86809            0.0           18.4      0.0      0.0      0.0      0.0 newOrder
		 1h0m37s    86809            0.0            1.9      0.0      0.0      0.0      0.0 orderStatus
		 1h0m37s    86809            0.0           17.2      0.0      0.0      0.0      0.0 payment
		 1h0m37s    86809            0.0            1.8      0.0      0.0      0.0      0.0 stockLevel
		E190318 15:14:39.322831 1 workload/cli/run.go:420  error in orderStatus: dial tcp 10.142.0.63:26257: connect: connection refused
		: signal: killed
	cluster.go:1626,tpcc.go:140,schemachange.go:375,test.go:1214: unexpected node event: 1: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/ec89b45cea7e8a6dd92a9bfd60e0cc06842e06d8

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=schemachange/indexrollback/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1197083&tab=buildLog

```
The test failed on release-19.1:
	cluster.go:1267,tpcc.go:130,cluster.go:1605,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1197083-schemachange-indexrollback-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=1h30m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		0.0      0.0 newOrder
		   7m11s    19245            0.0            0.1      0.0      0.0      0.0      0.0 orderStatus
		   7m11s    19245            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   7m11s    19245            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190325 14:38:02.921092 1 workload/cli/run.go:420  error in payment: dial tcp 10.142.0.71:26257: connect: connection refused
		   7m12s    63876            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   7m12s    63876            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   7m12s    63876            0.0            0.1      0.0      0.0      0.0      0.0 orderStatus
		   7m12s    63876            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   7m12s    63876            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190325 14:38:03.921211 1 workload/cli/run.go:420  error in payment: dial tcp 10.142.0.71:26257: connect: connection refused
		: signal: killed
	cluster.go:1626,tpcc.go:140,schemachange.go:375,test.go:1214: unexpected node event: 2: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/7bc9ea5fbe0c0082fdcfd408245a79c62b00edd4

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=schemachange/indexrollback/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1197065&tab=buildLog

```
The test failed on master:
	cluster.go:1267,tpcc.go:130,cluster.go:1605,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1197065-schemachange-indexrollback-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=1h30m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		   2.0      0.0      0.0      0.0      0.0 delivery
		  47m37s    59683            0.0           19.6      0.0      0.0      0.0      0.0 newOrder
		  47m37s    59683            0.0            2.0      0.0      0.0      0.0      0.0 orderStatus
		  47m37s    59683            0.0           18.4      0.0      0.0      0.0      0.0 payment
		  47m37s    59683            0.0            2.0      0.0      0.0      0.0      0.0 stockLevel
		E190325 15:06:53.700842 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.110:26257: connect: connection refused
		  47m38s    65628            0.0            2.0      0.0      0.0      0.0      0.0 delivery
		  47m38s    65628            0.0           19.6      0.0      0.0      0.0      0.0 newOrder
		  47m38s    65628            0.0            2.0      0.0      0.0      0.0      0.0 orderStatus
		  47m38s    65628            0.0           18.4      0.0      0.0      0.0      0.0 payment
		  47m38s    65628            0.0            2.0      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	cluster.go:1626,tpcc.go:140,schemachange.go:375,test.go:1214: unexpected node event: 1: dead

```

--- a/pkg/cmd/roachtest/schemachange.go
+++ b/pkg/cmd/roachtest/schemachange.go
@@ -325,7 +325,7 @@ func makeIndexAddTpccTest(numNodes, warehouses int, length time.Duration) testSp
 }
 
 func registerSchemaChangeCancelIndexTPCC1000(r *registry) {
-	r.Add(makeIndexAddRollbackTpccTest(5, 1000, time.Minute*90))
+	r.Add(makeIndexAddRollbackTpccTest(5, 1000, time.Minute*60))
 }
 
 // Creates an index and job, returning the job ID and a notify channel for
@@ -370,7 +370,7 @@ func makeIndexAddRollbackTpccTest(numNodes, warehouses int, length time.Duration
 	return testSpec{
 		Name:    fmt.Sprintf("schemachange/indexrollback/tpcc/w=%d", warehouses),
 		Cluster: makeClusterSpec(numNodes),
-		Timeout: length * 2,
+		Timeout: length * 3,
 		Run: func(ctx context.Context, t *test, c *cluster) {
 			runTPCC(ctx, t, c, tpccOptions{
 				Warehouses: warehouses,

